src/PostContent/PostContent.tsx

```
@@ -1,17 +1,12 @@
 import { Typography } from "hazel-ui/Typography";
 import { MDXRemote } from "next-mdx-remote";
 import Head from "next/head";
-import type { Post } from "../lib";
-import { Date } from "../ui";
-import { Layout } from "../wrappers";
 
-import type { MDXRemoteSerializeResult } from "next-mdx-remote";
+import type { Post } from "../lib/posts.js";
+import { Date } from "../ui/Atoms/Date.js";
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)